packages/manager/apps/web/client/app/domain/dns-modify/components/add-dns-forms/add-dns-form/add-dns-form.constants.js

```diff
@@ -0,0 +1,33 @@
+export const INTERNAL_DNS_PATTERN = {
+  EU: /^d?ns(?:\d{1,3})?\.ovh\.net/i,
+  CA: /d?ns\d{2}\.ovh\.ca/i,
+  TN: /d?ns\d\.tn\.ovh\.net/i,
+  REGISTRAR: /ns\d-registrar\.ovh\.net/i,
+  ANYCAST: /d?ns\d{3}\.anycast\.me/i,
+  PARKING: /parking\d\.ovh\.net/i,
+  WORLD: /d?ns\d?(?:\.(?:\w{3}|\w{3}-\d))?\.(?:cdn|interne)\.ovh\.net/i,
+  NG_GD: /d?ns\.ns-gd\.eu/i,
```

Comment:
**question:** Is it safe to put Internal DNS info like this on a public platform since the code is open-source.

Reply:
Seen with DNS team, no problem 
NB: the regex will be replaced by an API call when one is available to give us this information (for maintainability, no safety issue)

Reply:
Seen with DNS team, no problem
NB: the regex will be replaced by an API call when one is available to give us this information (for maintainability, no safety issue)